server/pulp/server/content/web/views.py

```diff
@@ -11,8 +11,9 @@
 from pulp.server.config import config as pulp_conf
 
 
-# Make sure all requested paths fall under this directory.
+# Make sure all requested paths fall under these directory.
```

Comment:
I guess it should be 'under this directory' or 'under these directories' If my English knowledge does not betray me :D